### PR TITLE
Enable Layout cops

### PR DESCRIPTION
**Pull request by gmcquistin**

### CODE REVIEW

File: .rubocop.yml [hunk 1]
```diff
@@ -12,7 +12,7 @@ Documentation:
   Enabled: false
 
 Layout/EmptyLines:
-  Enabled: false
+  Enabled: true
```
Comment (rdallasgray) [inline .rubocop.yml]: Just remove the declaration -- it's enabled by default.

File: .rubocop.yml [hunk 2]
```diff
@@ -23,15 +23,22 @@ Layout/EmptyLinesAroundModuleBody:
 Layout/EmptyLinesAroundClassBody:
   Enabled: false
 
-Layout/IndentHash:
-  Enabled: false
+Layout/ExtraSpacing:
+  Enabled: true
+
+Layout/TrailingBlankLines:
+  Enabled: true
```
Comment (rdallasgray) [inline .rubocop.yml]: Both of these are enabled by default. Check with https://rubocop.readthedocs.io before adding stuff.

Comment (gmcquistin) [reply .rubocop.yml]: should we copy the defaults into this file so we have a single source of truth?

Comment (rdallasgray) [reply .rubocop.yml]: I think that'd make it very hard to maintain, e.g. when they add, remove or change cops.

**Review by rdallasgray — APPROVED** [review]

File: .rubocop.yml (hunk 2 above)
Comment (gmcquistin) [reply .rubocop.yml]: hmm, yeah, just concerned about things changing under us - by falling back to their defaults I think we risk them changing (and us having to override that later on) - but that'll be rare hopefully!

**Pull request merged**